utils/activities.csv

```diff
@@ -195,10 +195,10 @@ Maak een lijstje van dingen die je kunt doen in plaats van roken.✍️ Bepaal d
 
 Dit is het einde van de oefening.",1,te voorkomen dat je gaat roken.
 23,Ontspanning,Leer hoe je kunt ontspannen.,"Deze oefening heet: 'Ontspanning'. 
-Zowel stoppen met roken als meer bewegen kan stressvol zijn en je een gespannen gevoel geven. Deze oefening helpt jouw spanning te verlagen. Progressieve spierontspanning is een goede manier om je lichaam te ontspannen. Wanneer we bewust onze spieren aanspannen en daarna ontspannen, helpt dit stress en spanning te verlagen. Je kan deze techniek in veel verschillende situaties toepassen. Bijvoorbeeld wanneer je zin hebt in een sigaret, tijdens een stressvolle werkdag, of na een inspannende work-out. Kijk de volgende video van 5 minuten, hierin word je stap voor stap door de oefening geleid: https://www.youtube.com/watch?v=PG1n-1cAV_0&ab_channel=ZiekenhuisGelderseValleiEde 
+Zowel stoppen met roken als meer bewegen kan stressvol zijn en je een gespannen gevoel geven. Deze oefening helpt jouw spanning te verlagen. Progressieve spierontspanning is een goede manier om je lichaam te ontspannen. Wanneer we bewust onze spieren aanspannen en daarna ontspannen, helpt dit stress en spanning te verlagen. Je kan deze techniek in veel verschillende situaties toepassen. Bijvoorbeeld wanneer je zin hebt in een sigaret, tijdens een stressvolle werkdag, of na een inspannende work-out. Kijk de volgende video van 5 minuten, hierin word je stap voor stap door de oefening geleid: https://www.Youtube.com/watch?v=PG1n-1cAV_0&ab_channel=ZiekenhuisGelderseValleiEde 
 
 Andere opties om stress en spanning te verminderen zijn:
-- Meditatie, mindfulness, of andere ademhalingstechnieken.🧘 Eenvoudige video’s hierover zijn te vinden op www.youtube.com. Zoek bijvoorbeeld op ‘5 minuten mindfulness meditatie’ of ‘begeleide beginners ademhalingsoefeningen’ of ‘mindfulness lichaamsscan’. Ook kan je een mindfulness of meditatie app downloaden (bijvoorbeeld de gratis app VGZ Mindfulness coach).
+- Meditatie, mindfulness, of andere ademhalingstechnieken.🧘 Eenvoudige video’s hierover zijn te vinden op www.Youtube.com. Zoek bijvoorbeeld op ‘5 minuten mindfulness meditatie’ of ‘begeleide beginners ademhalingsoefeningen’ of ‘mindfulness lichaamsscan’. Ook kan je een mindfulness of meditatie app downloaden (bijvoorbeeld de gratis app VGZ Mindfulness coach).
```

Comment:
I see that here the http-part is not there. This does not make a difference when it comes to the links working?

Reply:
No, there is no difference, they work the same